db/schema.rb

```diff
@@ -10,293 +10,293 @@
 #
 # It's strongly recommended that you check this file into your version control system.
```

Comment:
Huge diff because the indentations on every line differ. How strange for an autogenerated file

Reply:
Actually this made me realize we don't even need to add type column to input_files. Will change it to just a method and keep schema out of the diff